### DISCUSSION TITLE
Instance keypoints do not show up

### DISCUSSION THREAD

**Discussion by edofazza** [talmolab/sleap#798] [Help!] [answered]
I trained a model using 116 frames and I used it to predict the suggested frames obtained from the GUI. Obtained the video_name.predictions.slp I merged it to the project I used for creating the training package, as the documentation suggests. What happens now is that the predicted frames have small "Mean Score" values (e.g., 0.02) and when I check where the instance is positioned I cannot thing it.

<img width="1691" alt="Schermata 2022-06-16 alle 15 37 36" src="https://user-images.githubusercontent.com/46302551/174082316-abb7bcc2-df1b-4805-a80f-4c4897c720ce.png">

In the image, the number of points is 0/7 so it means that there are not points in it which it could make sense since the confidence is so low, but the true problem is when I create a new instance in the same image. What happens is that the newly create instance cannot be found, hence I cannot position correctly the keypoints.

<img width="1691" alt="Schermata 2022-06-16 alle 15 41 32" src="https://user-images.githubusercontent.com/46302551/174083016-48e97db5-c68c-49f9-8f38-8a02205629f0.png">

Is there a way to have the instances created in the center so to easily move them in the correct position?

Thanks

P.S.: this happens also when I create a new instance for frames which I do not have a prediction, sporadicly but it happens. Usually in this cases the trick is to go to a previously labeled frame, click on one of the keypoint and then create the instance again for the unlabeled frame. Unfortunately, this trick does not work for the predicted frames.

**Comment by roomrys** [comment, marked as answer]
Hi @edofazza,

It could be that your setting under View > Show Non-Visible Nodes is deselected. 

If that does not help, then a work around would be to [create instances](https://sleap.ai/guides/gui.html?highlight=force%20directed#labels:~:text=Instance%20Placement%20Method) using the Right Click + Force Directed method (or the Average method). The Default method will copy the predicted instance (which we are having trouble locating).

Let us know if either of these help.

Thanks,
Liezl

**Reply by edofazza** [reply]
Hi @roomrys,

View > Show Non-Visible Nodes is selected.

The Right Click + Force Directed method instead works perfectly fine and solves the issue.

Thank you
